docs/get-started.md

```diff
@@ -17,11 +19,11 @@ npm init -y
 then install Ocular as a devDependency.
 
 ```
-yarn add -D ocular-gatsby
+yarn add -D gatsby-theme-ocular
 ```
 or
 ```
-npm install ocular-gatsby --save-dev
+npm install gatsby-theme-ocular --save-dev
 ```
 
 ## Creating and running your Ocular website
```

Comment:
Are these instructions about running `ocular init` still correct?  Looks like ocular-config.js is still necessary but many options have been moved into gatsby-config.js.

I think these getting started instructions could probably use more elaboration. Specifically would be good to briefly cover the necessary config files that are created, along with links to relevant https://www.gatsbyjs.org/ doc pages.

Reply:
> Are these instructions about running ocular init still correct? 

I patched them up somewhat, but I think unless someone steps in and adds some test cases to `ocular init` we should probably remove it - most websites are created as a copy paste exercise, not with `ocular init`, and now ocular is just one of many plugins, better run something like `gatsby init`...